spring-integration-mongodb/src/main/resources/org/springframework/integration/mongodb/config/spring-integration-mongodb-4.3.xsd

```diff
@@ -45,9 +45,8 @@
 					<xsd:attribute name="query-expression" type="xsd:string">
 						<xsd:annotation>
 							<xsd:documentation>
-								SpEL expression which should resolve to a
-								String value identifying the
-								MongoDb Query. Also see 'query' attribute
+								SpEL expression which should resolve to a String query (please refer to the 'query' attribute) or a
+								BasicQuery object (e.q., query-expression="new BasicQuery('{''name'' : ''Bob''}').limit(2)").
```

Comment:
I think just `Query` is really enough.

I won't mind if someone decides to perform `TextQuery` as well.